src/components/Comments/GetComment/AllComments.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent } from "react";
+import { Box, Button, Typography, makeStyles } from "@material-ui/core";
+import TabViewIntroSection from "../../TabViewIntroSection";
+
+const getAllCommentEndpoint = ["GET /comments"];
+const getAllCommentHeading = "GET all comments";
+const getAllCommentSubtitle = "Need to get some comments?";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  buttonGroup: {
+    maxWidth: 800,
+  },
+  button: {
+    marginLeft: 10,
+    marginRight: 10,
+  },
```

Comment:
Given the fact that you need to create space between each button I would now suggest that a Grid be used, it has a much more elegant way.

Also, remember the rule every number being divisible by 8. A margin of 10 isn't.